pkg/k8s/apis/cilium.io/utils/utils.go

```diff
@@ -143,12 +119,99 @@ func ParseToCiliumRule(namespace, name string, r *api.Rule) *api.Rule {
 			}
 		}
 	}
+}
+
+func parseToCiliumEgressRule(namespace string, inRule, retRule *api.Rule) {
```

Comment:
Why not return the rule instead of passing it as a parameter to the function?

Reply:
See https://github.com/cilium/cilium/pull/2878#discussion_r173925037